taxi_service/settings.py

```diff
@@ -39,11 +38,13 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "debug_toolbar",
     "taxi",
 ]
 
 MIDDLEWARE = [
     "django.middleware.security.SecurityMiddleware",
+    "debug_toolbar.middleware.DebugToolbarMiddleware",
```

Comment:
The `DebugToolbarMiddleware` is included in the `MIDDLEWARE` list. Like the `debug_toolbar` app, this should only be used in development.